psi4/driver/p4util/python_helpers.py

```diff
@@ -30,12 +30,15 @@
 import re
 import sys
 import uuid
+import warnings
+
 import numpy as np
 import qcelemental as qcel
 
 from psi4 import core
 from psi4.driver import qcdb
 from . import optproc
+from .exceptions import *
```

Comment:
I think it would be nicer to enumerate the exceptions, rather than the wildcard.  It's kinda verbose, but it's also immediately obvious where each function came from.  You're far more updated on all the Pythonic conventions, so feel free to disregard.

Reply:
You're right -- I've enumerated them in more recent projects. And most of the time in psi it's only `ValidationError` anyways.